dag.yml

```diff
@@ -34,10 +42,13 @@ steps:
     - data://meadow/faostat/2022-02-10/faostat_metadata
   data://garden/faostat/2021-06-17/faostat_rl:
     - data://meadow/faostat/2021-06-17/faostat_rl
-  data://garden/explorers/2021/food_explorer:
-    - data://garden/faostat/2021-03-18/faostat_qcl
-    - data://garden/faostat/2021-04-09/faostat_fbsc
-    - data://garden/owid/latest/key_indicators
+  # TODO: uncomment!!! this is commented only to avoid running this step when I run command
```

Comment:
Is there a reason why pattern matching is applied to dependencies as well? This is what I get
```
(.venv) ➜  etl git:(enhance-wpp-population) etl data://garden/owid/latest/key_indicators --dry-run
Detecting which steps need rebuilding...
Running 3 steps:
1. data://garden/owid/latest/key_indicators...
2. data://garden/owid/latest/population_density...
3. data://garden/explorers/2021/food_explorer...
```
there's no easy way how to run only `data://garden/owid/latest/key_indicators` step which makes it a bit hard to debug